beacon-chain/cache/depositcache/deposits_cache.go

```diff
@@ -54,6 +53,7 @@ type DepositCache struct {
 	pendingDeposits   []*dbpb.DepositContainer
 	deposits          []*dbpb.DepositContainer
 	finalizedDeposits *FinalizedDeposits
+	depositMap        map[[48]byte][]*dbpb.DepositContainer
```

Comment:
I will argue that if we *always* only going to return the first deposit, we dont need a list for value. But i think extensibility is nice, maybe we'll need `DepositsByPubkey` one day